Cargo.toml

```diff
@@ -20,6 +20,7 @@ canonical = { version = "0.5", optional = true }
 canonical_derive = { version = "0.5", optional = true }
 anyhow = {version = "1", optional = true}
 dusk-bytes = "0.1"
+cfg-if = "1"
```

Comment:
I thought we agreed to never request only the major during a previous PR and Q&A.

Reply:
I said the exact oposite and cited [this reference in anyhow for example](https://docs.rs/anyhow/1.0.38/anyhow/#no-std-support) where you just import 1 and anything avobe that is always updated by cargo on each build.

Reply:
Then probably you had connection issue during the Q/A because we bring this issue and I also explained what is the rationale behind NOT using a single digit for versioning requirement. Since we have the stand down we can probably repeat there what was already said.

Also, about the example, I do not understand: all the example of `anyhow` shows the usage of `"1.0"` and not `"1"`, I just clicked on also on the link above and I see:
```
[dependencies]
anyhow = { version = "1.0", default-features = false }
```

So, major.minor.